CHANGELOG.md

```diff
@@ -14,9 +14,9 @@
 - FCOE Network
 - Hypervisor Cluster Profile
 - Hypervisor Manager
-- Interconnects
```

Comment:
keep interconnects and remove interconnect types, since there is no such resource

Reply:
its added now